### PR TITLE
remove bulk-require

### PR DESCRIPTION
**Pull request by dominictarr**
closes: https://github.com/ssbc/patchbay/issues/106
also, same in patchcore: https://github.com/ssbc/patchcore/pull/51

### CODE REVIEW

**Comment by dominictarr** [comment]
Also, while writing this, I figured out something about how depject is used in the topic/type naming style...
when a directory goes _four layers deep_ the last layer is a bunch of gives that is loaded as a `first` or `map`, for example: `./message/html/{decorate,layout,meta,render}/`

although, `./app/sync/initialise/` contains only `settings.js` but it doesn't return anything so it would also call the rest too (maybe some plugged in apps also have this?)

**Comment by ahdinosaur** [comment]
> when a directory goes _four layers deep_ the last layer is a bunch of gives that is loaded as a `first` or `map`, for example: `./message/html/{decorate,layout,meta,render}/`

yeah! @mmckegg took this technique to the next level with [`modject`](https://github.com/mmckegg/modject), where it matters if the last layer is a file or a directory.

**Comment by mixmix** [comment]
the initialise pattern is one I started exploring in ticktack. That it exists here is because I've started to port the idea into patchbay. 

Have been meaning to look at this @dominictarr , sorry it's taken time

**Comment by mixmix** [comment]
looks good. Needs `bulk-require` removing then ready to merge

**Comment by dominictarr** [comment]
fixed

**Comment by mixmix** [comment]
cool, I'll review this and get this in in the next release

File: package.json [hunk 1]
```diff
@@ -34,7 +34,6 @@
   },
   "homepage": "https://github.com/ssbc/patchbay#readme",
   "dependencies": {
-    "bulk-require": "^1.0.1",
```
Comment (mixmix) [inline package.json]: not to self (mix) remove bulkiy if it's not needed any more

**Pull request closed without merging**